libred/CMakeLists.txt

```diff
@@ -16,6 +16,32 @@ check_symbol_exists(_NSGetEnviron crt_externs.h HAVE_NSGETENVIRON)
 
 find_package(Threads REQUIRED)
 
```

Comment:
Thanks, did so. Although without the quotes; for some reason, cmake keeps the quotes, so for example I end up invoking `"usr/bin/cc"` which obviously doesn't exist.